app/upgrades.go

```diff
@@ -39,7 +40,7 @@ func (app *StrideApp) setupUpgradeHandlers() {
 	// no store upgrades
 	case "v3":
 		storeUpgrades = &storetypes.StoreUpgrades{
-			Added: []string{claimtypes.StoreKey},
+			Added: []string{claimtypes.StoreKey, authzkeeper.StoreKey},
```

Comment:
Is this a required step whenever a new module is added as part of an upgrade? Thanks for adding this, still learning how upgrades work

Reply:
this step is for new modules added to the upgrade, add module store to app multistore

Reply:
Thanks! Will keep this in mind in the future